pandas/conftest.py

```diff
@@ -1268,6 +1268,16 @@ def timedelta64_dtype(request):
     return request.param
 
 
+@pytest.fixture
+def fixed_now_ts():
+    """
+    Fixture emits fixed Timestamp.now()
+    """
+    return Timestamp(
+        year=2021, month=1, day=1, hour=12, minute=0, second=0, microsecond=0
```

Comment:
Per my previous suggestion, please make these _not_ zero. e.g. 4, 13, 22

Reply:
I fixed it